fcrepo-connector-file/pom.xml

```diff
@@ -21,6 +21,12 @@
       <groupId>javax.jcr</groupId>
       <artifactId>jcr</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.modeshape</groupId>
+      <artifactId>modeshape-jcr</artifactId>
+      <scope>provided</scope>
```

Comment:
I'm a little confused about this scope-- is the thought that the _container_ will provide this artifact (which clearly ain't the case...)?